Devtools mv3 samples (#938)
GoogleChrome/chrome-extensions-samples: api-samples/devtools/chrome-query/chrome-query.md

> # devtools

Reviewer: Should this file be named readme.md?

Reviewer: Sorry, my mistake, the naming convention is README.md, can you please update the name agian.

> This sample demonstrates using the devtools API to add a sidebar that displays the jQuery data associated with the selected DOM element.

Reviewer: Is it still worth demoing anything related to JQuery?

Reviewer 2: Long term I think a sample that doesn't use jQuery would be nice, but jQuery still exists, and so I don't think we necessarily need to change this given that our main goal is just having something that works in MV3. This feels like a useful sample for developers migrating even if the context is a bit less common these days.

## Overview

> The devtools window can be expanded with new elements created by the user. The extension contains html and js files that are injected into the devtools page to expand its functionality.

Reviewer 3: ```suggestion
This extension shows how DevTools can be expanded with new elements. It contains HTML and JavaScript files that are injected into the DevTools page to expand its functionality.
```
